packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-builder-ui-workspace/sortable-list.tsx

```diff
@@ -21,63 +10,77 @@ import {
 } from '@dnd-kit/core';
 import {
   SortableContext,
+  useSortable,
   verticalListSortingStrategy,
 } from '@dnd-kit/sortable';
-import Wizard from '../stage-wizard';
+import { CSS as cssDndKit } from '@dnd-kit/utilities';
+import type { SyntheticListenerMap } from '@dnd-kit/core/dist/hooks/utilities';
 
-const pipelineWorkspaceContainerStyles = css({
-  position: 'relative',
-  width: '100%',
-  height: '100%',
-});
-
-const pipelineWorkspaceStyles = css({
-  display: 'flex',
-  flexDirection: 'column',
-  width: '100%',
-  flexGrow: 1,
-});
+import Stage from '../../stage';
+import Wizard from '../../stage-wizard';
+import AddStage from '../../add-stage';
 
-const stageContainerStyles = css({
+const sortableItemStyles = css({
   display: 'flex',
   flexDirection: 'column',
 });
 
-type PipelineBuilderUIWorkspaceProps = {
-  stagesIdAndType: StageIdAndType[];
-  editViewName?: string;
-  onStageMoveEnd: (from: number, to: number) => void;
-  onStageAddAfterEnd: (after?: number) => void;
+export type SortableProps = {
+  style?: React.CSSProperties;
+  setNodeRef?: (node: HTMLElement | null) => void;
+  listeners?: SyntheticListenerMap;
```

Comment:
```suggestion
  style: React.CSSProperties;
  setNodeRef: (node: HTMLElement | null) => void;
  listeners: SyntheticListenerMap | undefined;
```

This is probably fine if we do it like this - as listeners are the only item that could be undefined, right?